modules/social_features/social_user/config/install/field.field.user.user.field_external_identifier.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
This is only going to be applied to a new installation, shouldn't we add this to config update?

Reply:
Actually we need to add hook update yes.